### PR TITLE
fallback rule of inlined url

### PR DESCRIPTION
**Pull request by valotas**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules
 test/fixtures/*.actual.css
+.idea/
```
Comment (MoOx) [inline .gitignore]: It's like .DS_Store & Thumbs.db.
You should ignore editor prefs in your global gitignore. I don't want to see everyone editor files here. I think you will thanks me later.

**Comment by MoOx** [comment]
I think most browser will download the last properties. Are you sure about what you are doing? Seems weird.

**Comment by valotas** [comment]
Well this is a fallback for IE only if I'm not mistaken. I used to use [`wro4j`](https://code.google.com/p/wro4j/) where you can find [this explanation](https://code.google.com/p/wro4j/wiki/FallbackCssDataUriProcessor).

File: .gitignore (hunk 1 above)
Comment (valotas) [inline .gitignore]: Well, it is a matter of taste I think. I do not expect everyone to have a proper global gitignore. But I can remove the commit if this is the only problem with the pull request.

**Comment by valotas** [comment]
Removed the `.gitignore` commit and added documentation

**Comment by MoOx** [comment]
This resource seems to be outdated or/and just wrong.
If you try this http://jsbin.com/rovuhapena/1/edit in a recent browser, you will see that this will create network request (that's not what we want). Tried on Firefox but I'm sure it will be the same on all browsers.
Only IE7- are not supporting datauri. If you need a fallback, provide an alternate css or just use old shitty class like showed here http://stackoverflow.com/questions/7481155/mshtml-fallback-for-data-uris

**Comment by MoOx** [comment]
Any comment? Or I will end up by closing this PR :/

**Comment by valotas** [comment]
Sorry, did not have much time for that outside my vacations :). I'll try to investigate this weekend!

**Comment by valotas** [comment]
Well it looks like you are right. So wro4j is either outdated or I (and
you) am doing something wrong on testing.

In any way the custom class solution that you suggest looks like a better
solution.

I think you can just close this.

**Pull request closed without merging**